client/web/src/repo/RepoHeader.tsx

```diff
@@ -204,7 +204,11 @@ export const RepoHeader: React.FunctionComponent<React.PropsWithChildren<Props>>
                 <Breadcrumbs
                     breadcrumbs={props.breadcrumbs}
                     location={props.location}
-                    className={classNames('justify-content-start', !props.forceWrap ? styles.breadcrumbWrap : '')}
+                    className={classNames(
+                        'justify-content-start',
+                        styles.breadcrumb,
+                        !props.forceWrap ? styles.breadcrumbWrap : ''
+                    )}
```

Comment:
```suggestion
                    className={classNames(
                        'justify-content-start pr-0',
                        !props.forceWrap ? styles.breadcrumbWrap : ''
                    )}
```
nit: as an alternative